Formula/x16-emulator.rb

```diff
@@ -0,0 +1,36 @@
+class X16Emulator < Formula
+  desc "Emulator for the Commander X16 8-bit computer"
+  homepage "https://github.com/commanderx16/x16-emulator"
+  url "https://github.com/commanderx16/x16-emulator/archive/r37.tar.gz"
+  sha256 "0c9dbf76ceb5668c32d3b9f04538cdfe891826eaac6bf57c68b13a0152ce15d2"
+  license "BSD-2-Clause"
+
+  depends_on "cc65" => :build
+  depends_on "sdl2"
+
+  resource "rom" do
+    url "https://github.com/commanderx16/x16-rom/archive/r37.tar.gz"
+    sha256 "6d33326ac95c12f09efa328bbee4e8a4c285a42599199d8d5a93d28b5801deef"
+  end
+
+  def install
+    resource("rom").stage do
+      system "make", "BUILD_DIR=build"
+      libexec.install "build/rom.bin"
+    end
+    system "make"
+    libexec.install "x16emu"
+    (bin/"x16emu").write <<~EOS
+      #!/bin/bash
+      cd #{libexec}
+      ./x16emu "$@"
+    EOS
+  end
+
+  test do
+    output = shell_output("#{bin}/x16emu --help", 1)
```

Comment:
Can we do something other than help?

Reply:
Not really, all the options are for running the emulator

Reply:
Can we make that fail reliably instead?

Reply:
Yeah we could use the "spawn, wait, then kill" method that some formulae use. We could use that output for matching

Reply:
@SeekingMeaning That’s something 👍